docs/README.md

```diff
@@ -1,80 +1,75 @@
-# Mineflayer
-
-[![NPM version](https://img.shields.io/npm/v/mineflayer.svg?color=success&label=npm%20package&logo=npm)](https://www.npmjs.com/package/mineflayer)
-[![Build Status](https://img.shields.io/github/actions/workflow/status/PrismarineJS/mineflayer/ci.yml.svg?label=CI&logo=github&logoColor=lightgrey)](https://github.com/PrismarineJS/mineflayer/actions?query=workflow%3A%22CI%22)
-[![Try it on gitpod](https://img.shields.io/static/v1.svg?label=try&message=on%20gitpod&color=brightgreen&logo=gitpod)](https://gitpod.io/#https://github.com/PrismarineJS/mineflayer)
-[![Open In Colab](https://img.shields.io/static/v1.svg?label=open&message=on%20colab&color=blue&logo=google-colab)](https://colab.research.google.com/github/PrismarineJS/mineflayer/blob/master/docs/mineflayer.ipynb)
-[![GitHub Sponsors](https://img.shields.io/github/sponsors/PrismarineJS)](https://github.com/sponsors/PrismarineJS)
-
-[![Official Discord](https://img.shields.io/static/v1.svg?label=OFFICIAL&message=DISCORD&color=blue&logo=discord&style=for-the-badge)](https://discord.gg/GsEFRM8)
+<h1 align="center">
+  <a href="https://github.com/PrismarineJS/mineflayer"><img src="https://github.com/SilkePilon/mineflayer/blob/master/PrismarineJS.png?raw=true" alt="PrismarineJS" width="170"></a>
+  <br>
+  Mineflayer
+  <br>
+</h1>
+
+<h4 align="center">🤖 Create Minecraft bots with a powerful, stable, and high level JavaScript <a href="api.md" target="_blank">API</a>, also usable from <a href="https://github.com/PrismarineJS/mineflayer/tree/master/examples/python" target="_blank">Python 🐍</a>.</h4>
+
+<p align="center">
```

Comment:
I agree it looks a bit better but not sure about having so much html here...
thoughts @extremeheat ?

Reply:
The discord button should be kept the same to direct people to it over them making Github issues